unity-environment/Assets/ML-Agents/Examples/Crawler/crawler POST PROCESSING.asset

```diff
@@ -0,0 +1,320 @@
+%YAML 1.1
```

Comment:
Can you explain what this does and how much it is needed ?

Reply:
post processing profile image profile (to adjust the colors). we can get rid

Reply:
removed